parsers/jscript.c

```diff
@@ -1800,6 +1806,26 @@ static bool parseMethods (tokenInfo *const token, const tokenInfo *const class,
 							makeJsTag (name, JSTAG_PROPERTY, NULL, NULL);
 				}
 			}
+			else if (isType (token, TOKEN_EQUAL_SIGN)
+					 || isType (token, TOKEN_SEMICOLON)
+					 || isType (token, TOKEN_CLOSE_CURLY)
+					 || (isType (token, TOKEN_IDENTIFIER)))
+			{
+				if (isType (token, TOKEN_EQUAL_SIGN))
+				{
+					readToken (token);
+					if (isKeyword (token, KEYWORD_function))
+						makeJsTag (name, JSTAG_FUNCTION, NULL, NULL);
```

Comment:
Inside a class it's gonna be a method, not a function (which is distinct in this parser, and will fix the scope kind for these fields)
```suggestion
						makeJsTag (name, JSTAG_METHOD NULL, NULL);
```

Reply:
@b4n, thank you.
I fixed and pushed the change. If the changes should be squashed into one for your review, let me know.